app/src/main/java/net/gsantner/markor/format/csv/Csv2MdTable.java

```diff
@@ -0,0 +1,160 @@
+/*#######################################################
+ *
+ *   Maintained 2023 by k3b
+ *   License of this file: Apache 2.0
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.csv;
+
+import static java.lang.Math.max;
+
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.ICSVParser;
+import com.opencsv.exceptions.CsvValidationException;
+
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple csv to md-table converter using OpenCsv implementation
+ * https://opencsv.sourceforge.net/ Licensed under Appache2
+ * <p>
+ * This file should not have dependencies to Android or Markor-Architecture.
+ */
+public class Csv2MdTable implements Closeable {
```

Comment:
Suggestion: Combine Csv2Mdable, CsvConfig, CsvMatcher to a single class optionally with inner classes. Don't see the need to scatter it

Reply:
I would prefer to not merge Csv2MdTable with CsvMatcher because of separation of concerns. (The same way as XxxSyntaxHighlighter and XxxTextConverter are seperated classes)

* CsvMatcher  is an implementation detail for CsvSyntaxHighlighter
* Csv2MdTable is an implementation detail for CsvTextConverter

Reply:
Then how about CsvMatcher --> to CsvSyntaxHighlighter and  
Csv2MdTable --> to CsvTextConverter  

?